.env

```diff
@@ -0,0 +1,2 @@
+METADATA_SOURCE_PATH="https://raw.githubusercontent.com/devx-services/connector-registry/dev/connectors-metadata.json"
+ARCHIVE_SOURCE_PATH="https://raw.githubusercontent.com/devx-services/connector-registry/dev/archive"
```

Comment:
We need to take out this file from the git control. We don't want to accidentally commit sensitive env vars.
Instead we can create a sample .env file (.env.sample) with an explanation what vars the project needs with some default example values.

Reply:
For example, https://github.com/magento-eqp/aio-template-registry-api/blob/main/.env.sample